projects/shikshalokam/src/lib/core-module/components/app-container/app-container.component.scss

```diff
@@ -0,0 +1,111 @@
+.dropdownDiv{
+    color: var(--app-black-color);
+    background-color: var(--white-color);
+    position: absolute ;
+    width:500px !important;
+    height:350px;
+    right: 8px;
+    top:35px;
+    z-index:105;
+    border-radius: 2px;
+    box-shadow: 0px 0px 3px 3px  rgba(0,0,0,0.08);
+    margin:20px;
+    padding: 20px 0px;
+    overflow-y: auto;
+
+}
+
+  .dropdownDiv:before {
+    content: "";
+    position: absolute;
+    top: -27px;  
+    right: 60px;
+    border-left: 13px solid transparent;
+    border-right: 13px solid transparent;
+    border-top: 13px solid transparent;
```

Comment:
minimise the styles
border: 13px solid transparent;
border-bottom-color: rgba(0,0,0,0.08)